### PR TITLE
Move descriptions

### PR DESCRIPTION
**Pull request by TheTiksiBranch**
Intended to be consistent with current move descriptions when possible. Short descriptions have been tested to make sure they fit (but if one isnt, or anything else needs addressing, tell me). Also made Noble Howl and Pyramiding Song authentic with approval.

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -1515,8 +1521,8 @@ let BattleMovedex = {
 		accuracy: 100,
 		basePower: 40,
 		category: "Special",
-		desc: "",
-		shortDesc: "",
+		desc: "Has a 100% chance to flinch the target. If this move hits a Pokemon not behind a Substitute, and if both the user and target are not already transformed or under the effect of Illusion, the user transforms into the opponent, copying its current stats, stat stages, types, moves, Ability, weight, gender, and sprite. The user's level and HP remain the same and each copied move receives only 5 PP, with a maximum of 5 PP each. The user can no longer change formes if it would have the ability to do so.",
+		shortDesc: "Hits first; user flinches and transforms into target.",
```
Comment (ghost) [inline mods/seasonal/moves.js]: target flinches, not the user

**Comment by HoeenCoder** [comment]
Yeah I think I managed to screw this up

**Comment by HoeenCoder** [comment]
Ok fixed!

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -109,8 +109,8 @@ let BattleMovedex = {
 		accuracy: 100,
 		basePower: 140,
 		category: "Special",
-		desc: "Has a 40% chance to paralyze the target. Lowers the user's spa, spd, and spe by 1 stage.",
-		shortDesc: "40% to paralyze target. Lowers user's spa, spd, spe.",
+		desc: "Has a 40% chance to paralyze the target. Lowers the user's SpA, SpD, and Spe by 1 stage.",
+		shortDesc: "40% to paralyze. Lowers user SpA, SpD, Spe.",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: user needs an s to become users (still fits)

File: mods/seasonal/moves.js [hunk 3]
```diff
@@ -136,8 +136,8 @@ let BattleMovedex = {
 		accuracy: true,
 		basePower: 0,
 		category: "Status",
-		desc: "",
-		shortDesc: "",
+		desc: "Sets Trick Room for 5 turns and raises the user's Special Attack by 1 stage.",
+		shortDesc: "User Sp. Atk +1; sets Trick Room.",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: Users

File: mods/seasonal/moves.js [hunk 4]
```diff
@@ -166,8 +166,8 @@ let BattleMovedex = {
 		accuracy: 100,
 		basePower: 70,
 		category: "Physical",
-		desc: "",
-		shortDesc: "",
+		desc: "If the target uses certain non-damaging moves this turn, the user steals that move to use itself. Fails if no move is stolen or if the user is under the effect of Sky Drop.",
+		shortDesc: "Steals status move; fails if it does not. Priority.",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: Fails if target is attacking. Steals foes move. Priority.

File: mods/seasonal/moves.js [hunk 5]
```diff
@@ -1439,7 +1443,7 @@ let BattleMovedex = {
 		basePower: 80,
 		category: "Physical",
 		desc: "Lowers the user's Defense, Special Defense and Speed by 1 stage.",
-		shortDesc: "Lowers the user's Def, SpD and Spe by 1.",
+		shortDesc: "Lowers the user's Def, Sp. Def. and Spe by 1.",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: Sp. Def (extra period)

**Review by HoeenCoder — COMMENTED** [review]
Because I used a cherry pick & update, there should be some new stuff w/o descs you can edit in now if you want (or later).

File: mods/seasonal/moves.js (hunk 1 above)
Comment (TheTiksiBranch) [reply mods/seasonal/moves.js]: the intent is to distribute "target" to both "flinches" and "transforms into". If it's not reading like that, though, it should be changed: "Hits first; user flinches target & transforms into it" which fits?

File: mods/seasonal/moves.js (hunk 5 above)
Comment (TheTiksiBranch) [reply mods/seasonal/moves.js]: Was supposed to be a comma: fixed

File: mods/seasonal/moves.js (hunk 4 above)
Comment (TheTiksiBranch) [reply mods/seasonal/moves.js]: Steals foe's move. Fails if target attacks. Priority.
(order for clarity, phrasing to extra-make clear that they're using a damaging move as opposed to merely acting)

File: mods/seasonal/moves.js (hunk 2 above)
Comment (TheTiksiBranch) [reply mods/seasonal/moves.js]: fixed

File: mods/seasonal/moves.js (hunk 3 above)
Comment (TheTiksiBranch) [reply mods/seasonal/moves.js]: fixed

File: mods/seasonal/moves.js (hunk 1 above)
Comment (ghost) [reply mods/seasonal/moves.js]: Ohhhh, now i see it. nvm

Comment (HoeenCoder) [reply mods/seasonal/moves.js]: Its confusing, I'd go with `Hits first; foe flinches; user transforms into target` (Make sure that fits)

Comment (TheTiksiBranch) [reply mods/seasonal/moves.js]: It does, and did. Also, I'll probably add the new ones soon.

**Pull request merged**